lib/blocks.php

```diff
@@ -76,6 +76,7 @@ function gutenberg_reregister_core_block_types() {
 				'post-author.php'             => 'core/post-author',
 				'post-comment.php'            => 'core/post-comment',
 				'post-comment-content.php'    => 'core/post-comment-content',
+				'post-comment-author.php'     => 'core/post-comment-author',
```

Comment:
Nitpick: let's move it above `post-comment-content.php` to keep the list alphabetical :grimacing: